device-server/src/main/kotlin/com/badoo/automation/deviceserver/data/XcuiTestExecutionConfig.kt

```diff
@@ -0,0 +1,21 @@
+package com.badoo.automation.deviceserver.data
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import java.nio.file.Path
+
+data class XcuiTestExecutionConfig(
+        @JsonProperty("app_name")
+        val appName: String,
```

Comment:
Looks like this property is unused. The only usages are for logging and tests.

Reply:
Done.
#36